docs/docker-image-details.md

```diff
@@ -19,6 +19,9 @@ docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino:0.6.0-incubating
 
 Changelog
 
+- apache/gravitino:0.6.1-incubating
+  - Based on Gravitino 0.6.1, you can know more information from 0.6.1 release notes.
```

Comment:
should be `0.6.1-incubating` not `0.6.1`

Reply:
Fixed.